pysat/instruments/templates/template_instrument.py

```diff
@@ -311,8 +311,13 @@ def load(fnames, tag=None, inst_id=None, custom_keyword=None):
 
     Note
     ----
-    Any additional keyword arguments passed to pysat.Instrument
-    upon instantiation are passed along to this routine.
+    - Any additional keyword arguments passed to `pysat.Instrument` upon
+      instantiation are passed along to this routine.
```

Comment:
```suggestion
    - Any additional keyword arguments passed to `pysat.Instrument` upon
      instantiation or via `load` that are defined above will be passed along to this routine.
```
Or something along those lines. Keyword support has been upgraded since the comment was originally written.